src/v/redpanda/application.h

```diff
@@ -129,6 +129,15 @@ class application {
         s = std::make_unique<Service>(std::forward<Args>(args)...);
         _deferred.emplace_back([&s] { s->stop().get(); });
     }
+
+    template<typename Service, typename... Args>
+    ss::future<>
+    construct_single_service_sharded(ss::sharded<Service>& s, Args&&... args) {
+        auto f = s.start(std::forward<Args>(args)...);
```

Comment:
this looks like it starts a full service not a single service

Reply:
Good catch. I'll fix this and maybe add an assert somewhere as well.